CMakeLists.txt

```
@@ -33,6 +33,8 @@ else()
   message(FATAL_ERROR "No ROS Distro set")
 endif()
 
+set(USES_DEPRECATD_EXPORT_API "dashing" "eloquent")
```

```suggestion
set(USES_DEPRECATD_EXPORT_API "ardent" "bouncy" "crystal" "dashing" "eloquent")
```
Please add the others as well since overall we are still building and testing against those. That will change soon though. Thanks!